communities/microgalaxy/lab/base.yml

```diff
@@ -18,6 +18,7 @@ subdomain: microGalaxy
 root_domain: usegalaxy.eu
 support: https://help.galaxyproject.org/
 general_support: https://galaxyproject.org/support/
+figure_microbiome_url: https://github.com/usegalaxy-eu/microgalaxy_paper_2025/blob/main/docs/figures/figure_1_B.png?raw=true
```

Comment:
Could you add to the README file of https://github.com/usegalaxy-au/galaxy-labs-engine to explain how to add image? Thanks

Reply:
here you go :) https://github.com/usegalaxy-au/galaxy-labs-engine/pull/34